src/models/auth/auth.service.ts

```diff
@@ -80,4 +81,24 @@ export class AuthService {
       msg: 'Logout success',
     };
   }
+  // { user: IResponse; isAdmin: boolean }
+  async findUserLogin(
+    token?: string,
```

Comment:
```suggestion
    token?: string,
```
Đang tìm user logged thì token phải là required chứ